test/test-service.js

```diff
@@ -106,7 +106,7 @@ function validateCreds(lang, service) {
 			, "naturalLanguageUnderstanding" , "personalityInsights" , "speechToText"
 			, "textToSpeech", "toneAnalyzer" , "visualRecognition"].includes(service)) {
 			// assert.fileContent(mappings_path, testUtils.PREFIX_SVC_BINDING_NAME + services);
-			assert.fileContent(mappings_path, `service_${SvcInfo[service]["customServiceKey"].replace(/-/g, '_')}`);
+			assert.fileContent(mappings_path, service.replace(/-/g, '_'));
 		}
 
 		if (lang == "SWIFT") {
```

Comment:
Is there a good reason not to test services for ce, I'm not that familiar with it? I don't see any, I'm guessing that's what's causing that tiny drop in code coverage by not testing [this line](https://github.com/IBM/generator-ibm-cloud-assets/pull/127/files?diff=unified&w=1#diff-f86a86aa4f3ceb09f975201170d902b5e392504e266e99cc5b227cfaeebaa2a0R147)